ctf/tests/lunatest.lua

```diff
@@ -0,0 +1,1142 @@
+-----------------------------------------------------------------------
```

Comment:
I had checked their repo just now and here are their pros and cons:
lunatest
* It is general testing framework
* It has 4 contributors
* It is developed since 2010 and last updated in the end of 2015. Looks like it is considered as finished.
* It is documented inside repository

shakedown
* It is "special" testing framework "only for minetest". Looks like it is a modpack for minetest.
* It has only 1 contributor
* It is actively developed since april
* It is documented on github wiki

I'm not sure if this is a good idea to test something inside a minetest. The whole idea is that you don't need to run MT to test anything.
Anyway, I'll try shakedown now and see how it works.